cmd/must-gather-clean/version.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	VERSION = "v0.0.1"
```

Comment:
can you add a script for bumping this too? and possibly also add this into prepare-release.

Reply:
Added the bump script. The `prepare-release.sh` script doesn't really deal with version information. So I don't want to bump the versions in there. But if that's going to be single entrypoint for building releases I will add it later.